sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/autorestTest.java

```diff
@@ -0,0 +1,364 @@
+            .policies(policies.toArray(new HttpPipelinePolicy[0]))
+            .build();
+
+        AzureTableImplBuilder azureTableImplBuilder = new AzureTableImplBuilder();
+        AzureTableImpl azureTable = azureTableImplBuilder
+            .pipeline(pipeline)
+            .url("/https://telboytrial.table.core.windows.net")
+            .buildClient();
+
+        try {
+            TablesImpl tables = azureTable.getTables();
+
+            StepVerifier.create(tables.deleteWithResponseAsync("ebTable", "ID23", Context.NONE))
+                .assertNext(response -> {
+                    System.out.println(response);
+                    Assertions.assertEquals(200, response.getStatusCode());
+                })
+                .expectComplete()
+                .verify();
+        } catch (Exception e) {
+            System.out.print(e);
+        }
+
+    }
+
+
+    static AzureTableImpl auth() {
+        final String connectionString = System.getenv("azure_tables_connection_string");
+
+        StorageConnectionString storageConnectionString
+            = StorageConnectionString.create(connectionString, new ClientLogger("tables"));
+
+        StorageAuthenticationSettings authSettings = storageConnectionString.getStorageAuthSettings();
+        TablesSharedKeyCredential sharedKeyCredential = new TablesSharedKeyCredential(authSettings.getAccount().getName(),
+            authSettings.getAccount().getAccessKey());
+
+        final List<HttpPipelinePolicy> policies = Arrays.asList(
+            new AddDatePolicy(),
+            new AddHeadersPolicy(new HttpHeaders().put("Accept", OdataMetadataFormat.APPLICATION_JSON_ODATA_MINIMALMETADATA.toString())),
+            new TablesSharedKeyCredentialPolicy(sharedKeyCredential),
+            new HttpLoggingPolicy(new HttpLogOptions()
+                .setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
+        );
+        final HttpPipeline pipeline = new HttpPipelineBuilder()
+            .policies(policies.toArray(new HttpPipelinePolicy[0]))
+            .build();
+        AzureTableImpl azureTable = new AzureTableImplBuilder()
+            .pipeline(pipeline)
+            .version("2019-02-02")
+            .url("https://telboytrial.table.core.windows.net")
+            .buildClient();
+        return azureTable;
+    }
+
+
+    @BeforeEach
+    void beforeTests() {
+        createTableHelper(tableA);
+        createTableHelper(tableB);
+
+        propertiesB.put("PartitionKey", "Store");
+        propertiesB.put("RowKey", "Boston");
+        propertiesB.put("Employees", "200");
+        insertEntityHelper(tableA, propertiesB);
+
+        Map<String, Object> propertiesA = new HashMap<>();
+        propertiesA.put("PartitionKey", "Store");
+        propertiesA.put("RowKey", "Atlanta");
+        propertiesA.put("Employees", "50");
+        insertEntityHelper(tableA, propertiesA);
+
+
+    }
+
+    @AfterEach
+    void afterTests() throws InterruptedException {
+        deleteTableHelper(tableA);
```

Comment:
You can call the method `deleteTable(...)`. Helper isn't necessary. DeleteTable is enough to describe the functionality of the method.